src/i-client-config.ts

```diff
@@ -104,10 +105,41 @@ export interface IPrecomputedClientConfig extends IBaseRequestConfig {
 }
 
 /**
- * Configuration for regular client initialization
- * @public
+ * Base options for the EppoClient SDK
  */
-export interface IClientConfig extends IBaseRequestConfig {
+export type IApiOptions = {
+  /**
+   * Your key for accessing Eppo through the Eppo SDK.
+   */
+  apiKey: string;
+
+  /**
+   * Override the endpoint the SDK uses to load configuration.
+   */
+  baseUrl?: string;
+
+  /**
+   * Force reinitialize the SDK if it is already initialized.
+   */
+  forceReinitialize?: boolean;
```

Comment:
As per my other comment, I propose to forbid re-initialization of standalone clients (because it's buggy and it's now possible to create new clients). While we can postpone the implementation of that, I believe that `buildAndInit` should not accept `forceReinitialize` already (as removing it later will be a major change)

Reply:
Agreed.
Pushed the forceReinit check to the soon-to-be-deprecated global static init method. Also moved this option to `ICompatibilityOptions` (maybe better named `LegacyOptions`?)